website/docs/vscode.md

```diff
@@ -454,6 +454,28 @@ Shows the extension output window on errors. Defaults to `false`.
 
 ---
 
+## Extension API
+
+Starting from version 0.3.41, Sorbet exports a public API.
```

Comment:
Is it worth clarifying that this is the extension, or is it obvious from contextf/

```suggestion
Starting from version 0.3.41, the Sorbet extension exports a public API.
```